src/providers/incoming-data/incoming-data.ts

```diff
@@ -461,10 +461,10 @@ export class IncomingDataProvider {
 
   private handleWalletConnectUri(uri: string): void {
     // Disable Wallet Connect
-    if (!this.appProvider.info._enabledExtensions.walletConnect) {
-      this.logger.warn('Wallet Connect has been disabled for this build');
-      return;
-    }
+    // if (!this.appProvider.info._enabledExtensions.walletConnect) {
```

Comment:
I think it's necessary for disabling WC from appConfig

Reply:
+1